Makefile

```diff
@@ -67,6 +67,7 @@ setup_application:
 	rm -f ${APP_DIR}/yarn.lock
 	(cd ${APP_DIR} && ${COMPOSER} config repositories.plugin '{"type": "path", "url": "../../"}')
 	(cd ${APP_DIR} && ${COMPOSER} config extra.symfony.allow-contrib true)
+	(cd ${APP_DIR} && ${COMPOSER} config minimum-stability dev)
```

Comment:
```suggestion
	cd ${APP_DIR} && ${COMPOSER} config minimum-stability dev
```

Reply:
We have got parenthesis everywhere at this moment in this Makefile. 
I suggest to improve that later because it's a details and the full setup process stills work.
It could be corrected later when we will have a fantastic monorepo and one PR will correct it on each plugins (one day...)